### PR TITLE
Adding example of multiple search gallery layouts usage

### PR DESCRIPTION
**Pull request by icazevedo**
#### What problem is this solving?

This PR adds the new gallery layout component to the store. It adds the list layout and allows the user to choose between them. It has different product-summaries for each layout and they vary by device (mobile/desktop/`1300px+`) in quantity and type.

#### How to test it?

[Workspace](https://icarosearch--storecomponents.myvtex.com/apparel---accessories)

#### Screenshots or example usage:

Screen Shot 2021-01-06 at 20 18 53

Screen Shot 2021-01-06 at 20 19 19

#### Related to / Depends on

Depends on [search-result#473](https://github.com/vtex-apps/search-result/pull/473)
Depends on [responsive-values#11](https://github.com/vtex-apps/responsive-values/pull/11)

### CODE REVIEW

**Review by klzns — APPROVED** [review]

File: store/blocks/pdp/product-specifications.jsonc [hunk 1]
```diff
@@ -79,5 +79,32 @@
     "props": {
       "message": "{specificationValue}"
     }
+  },
+  "product-specification-group#summary": {
```
Comment (klzns) [inline store/blocks/pdp/product-specifications.jsonc:83]: Ops, the blocks of this folder should only be related to pdp

File: styles/css/vtex.flex-layout.css [hunk 2]
```diff
@@ -96,3 +92,38 @@
 .flexRow--addToCartRow {
   padding-bottom: 1rem;
 }
+
+.flexRow--orderByAndSwitcher {
```
Comment (klzns) [inline styles/css/vtex.flex-layout.css:96]: Put this in a folder like `styles/css/search/vtex.flex-layout.css`

File: styles/css/vtex.product-summary.css [hunk 3]
```diff
@@ -23,12 +23,30 @@
   padding-bottom: 1rem;
 }
 
+.nameContainer--summaryListDesktop {
```
Comment (klzns) [inline styles/css/vtex.product-summary.css:26]: You can also move this changes to the new folder

File: styles/css/vtex.rich-text.css [hunk 4]
```diff
@@ -88,3 +88,18 @@
...
+
+.container--layout-option {
+  margin-left: 0.3em;
+}
+
+.paragraph--layout-option {
+  margin: 0;
+}
+
+.paragraph--summarySpecificationBullet {
```
Comment (klzns) [inline styles/css/vtex.rich-text.css:100]: same

File: styles/css/vtex.store-components.css [hunk 5]
```diff
@@ -115,3 +115,16 @@
   font-weight: bold;
   font-size: 30px;
 }
+
+.productBrandName--summaryListDesktop {
```
Comment (klzns) [inline styles/css/vtex.store-components.css:119]: same

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by klzns — APPROVED** [review]

File: store/blocks/product-summary/product-summary-list.jsonc [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+      "blockClass": "buyNowButton"
+    },
+    "children": [
+      "add-to-cart-button#buyNow"
+    ]
+  },
+
+  "add-to-cart-button#buyNow": {
+    "props": {
+      "text": "BUY NOW", //internacionalizar
```
Comment (igorbrasileiro) [inline store/blocks/product-summary/product-summary-list.jsonc:119]: This should be internationalized?!

**Review by igorbrasileiro — APPROVED** [review]
LGTM. Left one comment

File: store/blocks/product-summary/product-summary-list.jsonc (hunk 6 above)
Comment (victorhmp) [reply store/blocks/product-summary/product-summary-list.jsonc:119]: Yep, it could be! Using the Site Editor

**Review by victorhmp — APPROVED** [review]

File: store/blocks/product-summary/product-summary-list.jsonc (hunk 6 above)
Comment (icazevedo) [reply store/blocks/product-summary/product-summary-list.jsonc]: I ended up forgetting to remove it 😅

**Pull request merged**